playbooks/scale-up.yaml

```diff
@@ -0,0 +1,50 @@
+---
+# Set new number of nodes
+- hosts: masters
+  tasks:
+  - name: Get number of nodes
+    shell: oc get nodes | grep app-node -c
+    register: oc_old_result
+  - name: Set new number of nodes
+    set_fact:
+      # Number of nodes in the new configuration (incremented by 1 by default)
+      nodes: "{{ oc_old_result.stdout | int + 1 }}"
+  - name: Check that new value is higher
+    assert:
+      that: 'nodes | int > oc_old_result.stdout | int'
```

Comment:
got it, this protects the future playbooks execution from unexpected scale down while running scale-up play, right? And otherwise, the scale-down protects users from doing unexpected scale-up. LGTM now, thanks!